conans/client/command.py

```diff
@@ -1066,6 +1066,49 @@ def export(self, *args):
                                   lockfile_out=args.lockfile_out,
                                   ignore_dirty=args.ignore_dirty)
 
+    def export_all(self, *args):
+        """
+        Exports multiple recipes from a monorepo.
+
+        Each subdirectory of the given path will be considered a reciped folder.
+        """
+        parser = argparse.ArgumentParser(description=self.export.__doc__,
+                                         prog="conan export",
+                                         formatter_class=SmartFormatter)
+        parser.add_argument("path", help=_PATH_HELP)
+        parser.add_argument("reference", nargs='?', default=None,
+                            help="user/channel, Pkg/version@user/channel (if name "
+                                 "and version are not declared in the conanfile.py) "
+                                 "Pkg/version@ if user/channel is not relevant.")
+        parser.add_argument('-k', '-ks', '--keep-source', default=False, action='store_true',
```

Comment:
Is the --keep-source very necessary for this use case? Maybe not? I would try to simplify and remove it.